core/src/main/java/org/fao/geonet/kernel/GeonetworkDataDirectory.java

```diff
@@ -162,6 +162,9 @@ private Path lookupProperty(JeevesServlet jeevesServlet, ServiceConfig handlerCo
                     if (jeevesServlet != null) {
                         value = jeevesServlet.getInitParameter(keyToUse);
                     }
+                    if ( (value == null) && (jeevesServlet.getServletContext() != null) ){
```

Comment:
If you look at line 162  it is suggesting that jeevesServlet could be null.
`if (jeevesServlet != null) {`

So maybe this should be 
`if ( value == null && jeevesServlet != null && jeevesServlet.getServletContext() != null ){`

otherwise there could be a nullpointer error if jeevesServlet is null

Reply:
fixed - that should only happen in test cases...